pb/payment.pb.go

```diff
@@ -1,18 +1,17 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.20.1-devel
+// 	protoc-gen-go v1.20.0-devel
```

Comment:
Can we use the same latest version, otherwise we will be pushing these changes overwriting each other all the time?